app/models/conversation.rb

```diff
@@ -40,6 +40,7 @@
 #  index_conversations_on_contact_inbox_id            (contact_inbox_id)
 #  index_conversations_on_first_reply_created_at      (first_reply_created_at)
 #  index_conversations_on_id_and_account_id           (account_id,id)
+#  index_conversations_on_identifier_and_account_id   (identifier,account_id)
```

Comment:
This change comes from the annotate gem. I'm not sure why it wasn't in the develop branch already. It's unrelated to this though.